client/my-sites/domains/domain-management/list/style.scss

```diff
@@ -153,7 +153,7 @@
 				padding-inline: 21px;
 
 				@include break-mobile {
-					padding-inline: 48px;
+					padding-inline: 24px;
 				}
```

Comment:
Doing this only on large displays is interesting ... because the mobile layout of v2 always uses 24px, even on mobile. Doesn't mean that we want the mobile layout of v1 to also use 24px?

Reply:
Made `24px` as default by https://github.com/Automattic/wp-calypso/pull/104490/commits/f754a6acf47487d5022e57b726dfd6c41729e56a